static/app/utils/replays/projectSupportsReplay.tsx

```diff
@@ -0,0 +1,7 @@
+import type {MinimalProject} from 'sentry/types';
+
+function projectSupportsReplay(project?: MinimalProject) {
+  return Boolean(project?.platform?.includes('javascript'));
```

Comment:
I just discovered `static/app/data/platformCategories.tsx` and the export `frontend`  which might be a more maintainable thing to check against.

Reply:
would dart or unity be supported though? those don't sound like web platforms

Reply:
I was thinking we could also augment the file and add our own list. Josh has worked on a prototype sdk for native iOS, so a list of things could be better long-term compared to using `.include`